lib/Dialect/Moore/MooreOps.cpp

```diff
@@ -385,19 +386,34 @@ void AssignedVarOp::getAsmResultNames(OpAsmSetValueNameFn setNameFn) {
 
 void ConstantOp::print(OpAsmPrinter &p) {
   p << " ";
-  p.printAttributeWithoutType(getValueAttr());
+  const auto &value = getValue();
+  if (value == FVInt(1, 0)) {
+    p << "false";
+  } else if (value == FVInt(1, 1)) {
+    p << "true";
```

Comment:
Yeah good point. I think my initial motivation to add them is to avoid having single-bit 0 and 1 printed as `0` and `-1`. But instead we could also just not use negative notation for single-bit numbers. That would probably make more sense :grin: